tests/test_cli/ex2.abci.out

```diff
@@ -11,14 +11,14 @@
 -> code: OK
 
 > check_tx 0x00
--> code: BadNonce
+-> code: 2
```

Comment:
don't we want to write some `humanCode` helper for that?

Reply:
We do, but it's not really easy to do in general since the codes will be the responsibility of the app. We have an example code now in `example/code` but the `abci-cli` is more general than that so it shouldn't print `BadNonce` when it might be talking to an app for which `2` means `BadGong` or whatever.

Reply:
We should keep thinking about how to solve this but I don't have a good solution yet

Reply:
got it. I was talking more about providing an example of such wrapper (possibly inside an app) to devs.

Reply:
Opened an issue: https://github.com/tendermint/abci/issues/146

Reply:
> got it. I was talking more about providing an example of such wrapper (possibly inside an app) to devs.
 
That'd be nice but I still don't see how we'd be able to use it here since `abci-cli` is too general purpose (it doesn't load an app in or anything). Good think we have `log`